### PR TITLE
TC_02.03.019 | Dashboard > Welcomed and explanation message

### PR DESCRIPTION
**Pull request by Stepwow**
https://trello.com/c/ybwo7aXd/315-tc0203019-dashboard-welcomed-and-explanation-message

### CODE REVIEW

**Comment by Sensation3000** [comment]
Hi, I will approve since it's  your first PR, but next time use fixtures please

**Review by Sensation3000 — APPROVED** [review]

**Review by ElenaDetusheva — COMMENTED** [review]
If I'm not mistsken , 1"it" is 1 TC
it('name of TC like in Trello').
 Something like this :it('TC_02.03.019 | Dashboard.........')

File: cypress/e2e/welcomeText.cy.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+///<reference types ="cypress"/>
+
+describe('welcomeText', () => {
+    
+    it('verify Welcome to Jenkins massage', function () {
```
Comment (S-Lanchik) [inline cypress/e2e/welcomeText.cy.js:5]: Add number of your TC

File: cypress/e2e/welcomeText.cy.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        cy.get('.empty-state-block h1').should('be.visible').and('have.text', 'Welcome to Jenkins!')
```
Comment (S-Lanchik) [inline cypress/e2e/welcomeText.cy.js:6]: Put 'Welcome to Jenkins!' to fixtures

File: cypress/e2e/welcomeText.cy.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    }) 
+    it('verify Explenation text', function () {
+        cy.get('.empty-state-block p').should('be.visible').and('have.text', 'This page is where your Jenkins jobs will be displayed. To get started, you can set up distributed builds or start building a software project.')
+    })
```
Comment (S-Lanchik) [inline cypress/e2e/welcomeText.cy.js:10]: Make the same corrections here

**Review by AlYeMax — APPROVED** [review]

File: cypress/e2e/welcomeText.cy.js (hunk 1 above)
Comment (Stepwow) [reply cypress/e2e/welcomeText.cy.js]: I made changes. Could you check it again?

File: cypress/e2e/welcomeText.cy.js (hunk 2 above)
Comment (Stepwow) [reply cypress/e2e/welcomeText.cy.js]: I made changes. Could you check it again?

File: cypress/e2e/welcomeText.cy.js (hunk 3 above)
Comment (Stepwow) [reply cypress/e2e/welcomeText.cy.js:8]: I made changes. Could you check it again?

**Review by Rom-T — APPROVED** [review]

**Review by S-Lanchik — APPROVED** [review]

**Pull request merged**